core/container/externalbuilder/copy_test.go

```diff
@@ -19,23 +20,35 @@ import (
 
 var _ = Describe("copy", func() {
 	var (
-		logger                             *flogging.FabricLogger
-		srcRootDir, srcSubDir, destRootDir string
-		srcRootFile, srcSubFile            *os.File
-		err                                error
+		logger                  *flogging.FabricLogger
+		srcRootDir, destRootDir string
+		err                     error
 	)
 
 	BeforeEach(func() {
 		srcRootDir, err = ioutil.TempDir("", "copy-test-")
 		Expect(err).NotTo(HaveOccurred())
-		srcSubDir, err = ioutil.TempDir(srcRootDir, "sub-")
+
+		err = ioutil.WriteFile(filepath.Join(srcRootDir, "file-in-root.txt"), []byte{}, os.ModePerm)
 		Expect(err).NotTo(HaveOccurred())
-		srcRootFile, err = ioutil.TempFile(srcRootDir, "file-")
+
+		err = os.Symlink("file-in-root.txt", filepath.Join(srcRootDir, "symlink-in-root.txt"))
 		Expect(err).NotTo(HaveOccurred())
-		srcSubFile, err = ioutil.TempFile(srcSubDir, "subfile-")
+
+		err = os.Mkdir(filepath.Join(srcRootDir, "subdir"), os.ModePerm)
```

Comment:
The repeated use of `filepath.Join(srcRootDir, "subdir")` means we should probably have a var.